doc/faq.rst

```diff
@@ -116,6 +116,14 @@ improvements, if any, with benchmarks and/or plots. It is expected that the
 proposed algorithm should outperform the methods that are already implemented
 in scikit-learn at least in some areas.
 
+Inclusion of a new algorithm speeding up an existing model is easier if:
+
+- It does not introduce new hyper-parameters (as it makes the library
+  more future-proof)
+- It is easy to document clearly when the contribution improves the speed
+  and when it does not.
```

Comment:
this second point might benefit from an example "(e.g. when n_features >> n_samples)"

might want periods at the end of every list item for consistency